WooCommerce/Classes/ViewRelated/ReusableViews/OverlayMessageView.xib

```diff
@@ -17,7 +15,7 @@
             <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
             <subviews>
                 <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" distribution="equalSpacing" alignment="center" spacing="25" translatesAutoresizingMaskIntoConstraints="NO" id="2w1-dg-Nr4">
-                    <rect key="frame" x="0.0" y="213.5" width="375" height="240.5"/>
+                    <rect key="frame" x="0.0" y="211.5" width="375" height="244.5"/>
```

Comment:
Also here, I don't know if this is a wanted change or an automatic modification of Xcode.

Reply:
(same as below)